### DISCUSSION TITLE
因使用错误型号的镜像 armbian-installl 导致的问题

### DISCUSSION THREAD

**Discussion by Kloping** [ophub/amlogic-s9xxx-armbian#1545] [Amlogic]
因使用错误型号的镜像 armbian-installl 
目前 无法进入 armbian系统 无法从U盘启动 卡在开机画面

请问 有没有办法急救

**Comment by ophub** [comment]
线刷安卓系统，恢复成安卓

**Reply by Kloping** [reply]
已经刷入 android_tv_e900v22e_cm211-1_201-1_s905l3b 但是 无法开启adb远程
